index.html

```diff
@@ -1050,12 +1050,12 @@
 <div class="practice">
 
 <p><span class="practicelab" id="user-agent-data-sharing">
-User agents should only share and present information about the user that
-the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
+User agents should only share information about the user that the user can reasonably
+anticipate being shared, and which directly relate to the user's goals.</span></p>
```

Comment:
@dmarti suggested that we think about the user's goals in terms of what they would do themself if they had enough time. So something like

```suggestion
anticipate being shared, and which the user would do themself if they had complete information and enough time.</span></p>
```

or s/enough time/could do as many things each second as their computer can/. There's still something missing about how the UA guesses what the user would do themself: is that what a reasonable observer would guess the user would do themself, or is the "should" enough to capture that any agent has to guess about their principal's wishes?

@hober was worried that even given unlimited time, the user might not bother (her example) reading the T&Cs or (my example) telling an advertiser where they saw the ad that got them to buy a thing because it's more fun to just go watch a movie. That's probably fine ... if they don't actually have enough of a goal to fund their websites that they'd dedicate an insignificant amount of time to it, it's reasonable to think their UA shouldn't be doing it either.

Reply:
People who make well-informed decisions to buy legitimate products are fairly likely to answer reasonable questions from the seller about "where did you see our ad?" But most people would not log and report on all the ads they see, good and bad.